src/utils/converters.ts

```diff
@@ -0,0 +1,68 @@
+import { letter } from './emotes.json';
+
+export function toDefault(content: string) {
+	// it is awful, i need suggestions for a better way to do this
+	// (arr be the content with each letter as an item)
+	content = content.replace(/\n/g, '');
+	content = content.replace(/>/g, '> ');
+	const arr = content.split(' ');
+	arr.splice(-1);
+
+	const table: Record<number, string> = Object.create(null);
+	for (let i = 0; i < 6; i++) table[i + 1] = '';
+
+	let convertedLetters = '';
+	arr.forEach(emoji => {
+		if (Object.values(letter['yellow']).includes(emoji)) {
+			convertedLetters += '🟨 ';
+		} else if (Object.values(letter['green']).includes(emoji)) {
+			convertedLetters += '🟩 ';
+		} else if (Object.values(letter['gray']).includes(emoji)) {
+			convertedLetters += '⬛ ';
+		}
+	});
+
+	const tableArr = convertedLetters.split(' ');
+	tableArr.splice(-1);
+
+	const lineLength = tableArr.length / 5;
+	for (let i = 0; i < lineLength; i++) {
+		table[i + 1] = tableArr.splice(0, 5).join('');
+	}
+
+	return Object.values(table).map(line => line).join('\n');
+}
+
+export function toEmoji(content: string, correct: string) {
+	const [contentArr, correctArr] = [content.split(''), correct.split('')];
+
+	const usedLetters = [];
+	const emojiWord: Record<number, string> = Object.create(null);
+	for (let i = 0; i < 6; i++) emojiWord[i + 1] = '';
+
+	for (let i = 0; i < contentArr.length; i++) {
+		usedLetters.push(contentArr[i]);
+
+		if (contentArr[i] === correctArr[i]) {
+			emojiWord[i + 1] = letter.green[contentArr[i] as typeof alphabetType];
```

Comment:
I think these `contentArr[i]` should be moved to the top of the loop, since it's used like 4 times

Reply:
True, nice catch.